pkg/config/config.go

```diff
@@ -23,92 +26,52 @@ type (
 		Path   string `json:"path" yaml:"path" toml:"path"`
 		OutDir string `json:"out_dir" yaml:"out_dir" toml:"out_dir"`
 
-		Defaults       Defaults                  `json:"defaults" yaml:"defaults" toml:"defaults"`
-		ExecutionUnits map[string]*ExecutionUnit `json:"execution_units,omitempty" yaml:"execution_units,omitempty" toml:"execution_units,omitempty"`
-		StaticUnit     map[string]*StaticUnit    `json:"static_unit,omitempty" yaml:"static_unit,omitempty" toml:"static_unit,omitempty"`
-		Exposed        map[string]*Expose        `json:"exposed,omitempty" yaml:"exposed,omitempty" toml:"exposed,omitempty"`
-		Persisted      map[string]*Persist       `json:"persisted,omitempty" yaml:"persisted,omitempty" toml:"persisted,omitempty"`
-		PubSub         map[string]*PubSub        `json:"pubsub,omitempty" yaml:"pubsub,omitempty" toml:"pubsub,omitempty"`
-		Config         map[string]*Config        `json:"config,omitempty" yaml:"config,omitempty" toml:"config,omitempty"`
-	}
-	Expose struct {
-		Type                   string                 `json:"type" yaml:"type" toml:"type"`
-		ApiType                string                 `json:"api_type" yaml:"api_type" toml:"api_type"`
-		ContentDeliveryNetwork ContentDeliveryNetwork `json:"content_delivery_network,omitempty" yaml:"content_delivery_network,omitempty" toml:"content_delivery_network,omitempty"`
-		InfraParams            InfraParams            `json:"pulumi_params,omitempty" yaml:"pulumi_params,omitempty" toml:"pulumi_params,omitempty"`
-	}
-
-	Persist struct {
-		Type        string      `json:"type" yaml:"type" toml:"type"`
-		InfraParams InfraParams `json:"pulumi_params,omitempty" yaml:"pulumi_params,omitempty" toml:"pulumi_params,omitempty"`
+		Defaults            Defaults                  `json:"defaults" yaml:"defaults" toml:"defaults"`
+		ExecutionUnits      map[string]*ExecutionUnit `json:"execution_units,omitempty" yaml:"execution_units,omitempty" toml:"execution_units,omitempty"`
```

Comment:
Yeah i dont disagree, but that seems like it would be a fair amount more work, so id like to defer that until we maybe add another kind of have a reason to go move it out